tools/cldr-unittest/src/org/unicode/cldr/unittest/TestCheckDisplayCollisions.java

```diff
@@ -78,15 +78,46 @@ public void testInheritance() {
 
         CheckDisplayCollisions cdc = new CheckDisplayCollisions(factory);
         cdc.setEnglishFile(CLDRConfig.getInstance().getEnglish());
-        
+
         CLDRFile frResolved = factory.make("fr", true);
         checkFile(cdc, fr, frResolved);
 
         CLDRFile frCaResolved = factory.make("fr_CA", false);
-        checkFile(cdc, frCA, frCaResolved, 
+        checkFile(cdc, frCA, frCaResolved,
             scorpioEmoji, ukRegion);
     }
-    
+
+    public void testCountNormalization() {
+        final String testLocale = "pt";
+        final String unitPattern1 = "//ldml/units/unitLength[@type=\"long\"]/unit[@type=\"graphics-dot\"]/unitPattern[@count=\"one\"]";
```

Comment:
Must test the three cases:
 "//ldml/units/unitLength[@type=\"long\"]/unit[@type=\"graphics-dot\"]/unitPattern[@count=\"one\"]"
 "//ldml/units/unitLength[@type=\"long\"]/unit[@type=\"graphics-dot\"]/unitPattern[@count=\"other\"]" // different count as # 1. MUST NOT COLLIDE WITH #1
"//ldml/units/unitLength[@type=\"long\"]/unit[@type=\"length-point\"]/unitPattern[@count=\"one\"]"; // different unit as # 1. MUST COLLIDE WITH #1

Reply:
And that is with all 3 having the same value.

Reply:
Also check to make sure that case="nominative" and case="accusative" not collide when they paths are otherwise identical and have the same value.

Reply:
OK, I'll add those tests and make another PR.